editor.planx.uk/src/@planx/components/PlanningConstraints/Public.tsx

```diff
@@ -216,7 +216,7 @@ function Component(props: Props) {
               _overrides,
               "planningConstraints.action": userAction,
               _nots: notsAfterOverrides,
-              ...intersectingConstraintsAfterOverrides,
+              ...(intersectingConstraintsAfterOverrides[props.fn]?.length === 0 ? undefined : intersectingConstraintsAfterOverrides),
```

Comment:
Nice!

Reply:
Good catch with this one !

Reply:
Sorry @DafyddLlyr - actually second guessing this now - can you please test on this flow https://3510.planx.pizza/opensystemslab/digital-land-api-search-test/preview

When undefined instead of empty array, then any future questions asking about `property.constraints.planning` are NOT going to be automated because the passport doesn't have the key at all / thinks it's seeing it for the first time!

Reply:
Okay full circle :cyclone: I think this is actually consistent with current production behavior for sites with no constraints & good to go. Agree that at minimum, `undefined` is the most easy-to-reason-about behavior and semantically correct for this journey! 

Going to merge this since `handleOverrides` has quite solid test coverage, but then carve out some more time in a new PR to really go through edge cases and write more tests :+1: